spec/features/index_page_spec.rb

```diff
@@ -1,6 +1,7 @@
 RSpec.describe '/content' do
   include GdsApi::TestHelpers::ContentDataApi
   include TableDataSpecHelpers
+  let(:metrics) { %w[pageviews unique_pageviews number_of_internal_searches feedex_comments] }
```

Comment:
`metrics` is only use within the test. Would you mind moving it there?
I am also curious about why we need to specify them at this point. Shouldn't we have a default?